### ISSUE TITLE
Add support for cross cluster search

### ISSUE THREAD

**Issue by elfisher** [opensearch-project/piped-processing-language#6]
When using the Workbench to query with PPL, it doesn't appear to support cross cluster search. It would be great to add support for cross cluster search.

**Comment by elfisher** [comment]
@anirudha do you know what it would take to make this work? Is this also a gap in SQL? I can create an issue there too.

**Comment by jackychanel** [comment]
Yep ! feature will be very useful  as we are dealing with several regions using CCS.
Seems to me a must have feature when using PPL !
